src/PostHog.AI/PostHog.AI.csproj

```
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Probably should target the same thing that the [core PostHog lib does](https://github.com/PostHog/posthog-dotnet/blob/main/src/PostHog/PostHog.csproj#L4):

```suggestion
    <TargetFrameworks>netstandard2.1;net8.0</TargetFrameworks>
```

---

totally! fixed